sdx-encrypted-submitter.go

```diff
@@ -111,6 +140,7 @@ func sendToRabbit(url string, exchange string, routingKey string, msgBody []byte
 		return err
 	}
 
+	var headers = amqp.Table{"tx_id": txID}
```

Comment:
No need to pre-declare `headers` - can just use `amqp.Table{}` directly when assigning the value in the struct

Reply:
Done - hang over from debugging . Good catch thanks.